finagle-redis/src/main/scala/com/twitter/finagle/redis/protocol/commands/PubSub.scala

```diff
@@ -0,0 +1,96 @@
+package com.twitter.finagle.redis.protocol
+
+import org.jboss.netty.buffer.ChannelBuffer
+import org.jboss.netty.buffer.ChannelBuffers
+import com.twitter.finagle.redis.ClientError
+import com.twitter.finagle.redis.protocol.Commands._
+import com.twitter.finagle.redis.util._
+
+abstract class SubscribeCommand extends RedisMessage {
+  def command: String
+  def listener: SubscribeListener
+}
+
+trait SubscribeListener {
```

Comment:
let's move this closer to the implementation–I had to do quite a bit of looking to find it.  also, let's not call it SubscribeListener, since we use Listeners for things that listen for connections and then respond to requests (ie Netty3Listener).  This is similar, but not exactly the same thing.  Maybe call it something different?  Handler?